### ISSUE TITLE
backup watchlist before each write

### ISSUE THREAD

**Issue by Edmond120** [aziz512/youtube-tracker#21]
If watchlist is "watchlist.ini" then before each write, "watchlist.ini" should be copied to "watchlist.ini.bak"

**Comment by aziz512** [comment]
What's the purpose of this? Backup in case there's an IO failure when writing?

**Comment by Edmond120** [comment]
Yeah, IO failure or in the case of a bug that causes invalid data to be written to the watchlist.

**Comment by aziz512** [comment]
👍 Good to have but lower in priority compared to main functionality.
